common.sh

```diff
@@ -117,7 +117,13 @@ fi
 if [ ! -d "$WORKING_DIR" ]; then
   echo "Creating Working Dir"
   sudo mkdir "$WORKING_DIR"
+  sudo chown "${USER}:${USER}" "$WORKING_DIR"
+  chmod 755 "$WORKING_DIR"
 fi
-sudo chown "${USER}:${USER}" "$WORKING_DIR"
-chmod 755 "$WORKING_DIR"
 
+if [ ! -d "$IRONIC_DATA_DIR" ]; then
+  echo "Creating Ironic Data Dir"
+  sudo mkdir "$IRONIC_DATA_DIR"
```

Comment:
`install` could do this with just one command, what's the permission changes for anyway? Is the default not good enough?

```console
install -d -o ${USER} -g ${USER} -m 0755 $IRONIC_DATA_DIR
```

Reply:
IIRC the dir needs to be world-readable for the httpd/tftp files to be readable, but yeah we could probably simplify this, for now I just copied what we already have for WORKING_DIR

Reply:
Was just curious, thanks